project/profiles/capacity4more/behat/features/group_dashboard.feature

```diff
@@ -40,3 +40,15 @@ Feature: Group dashboard
     And   I am logged in as user "president"
     And   I visit the dashboard of group "Movie Popcorn Corner"
     Then  I should see the group dashboard without quick post form
+
+  @api
+  Scenario: Check Invite a member link as group owner
+    Given I am logged in as user "mariecurie"
+    When  I visit the dashboard of group "Movie Popcorn Corner"
+    Then  I should see the link "Invite a member"
```

Comment:
@ybaras 
good work with the tests !

can you change it a little bit so the test will click the button and make sure we got the the invite form ?
thanks